content/content-strategy/audit-content/plan-your-audit/toc.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: nav/sections
+sections:
+  - Talk about team benefits
+  - Align your goals with agency priorities
+  - Show the costs and efficiencies
+  - Baseline your site
+  - Build empathy with feedback
```

Comment:
Does not match corresponding section heading. Will be a broken link

Reply:
Trisha fixed